src/scancode/cli.py

```diff
@@ -223,25 +234,11 @@ def reindex_licenses(ctx, param, value):
 number of files processed. Use --verbose to display file-by-file progress.
 '''
 
-
 class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
-
-formats = ('json', 'json-pp', 'html', 'html-app', 'spdx-tv', 'spdx-rdf')
-
-def validate_formats(ctx, param, value):
-    """
-    Validate formats and template files. Raise a BadParameter on errors.
-    """
-    value_lower = value.lower()
-    if value_lower in formats:
-        return value_lower
-    # render using a user-provided custom format template
-    if not os.path.isfile(value):
-        raise click.BadParameter('Invalid template file: "%(value)s" does not exist or is not readable.' % locals())
-    return value
+    options = [ op for op in scan_proper_plugins.hook.add_cmdline_option(scan_output_plugins=scan_output_plugins) if isinstance(op, click.Option) ]
```

Comment:
there is an extra space after `[` . is there a possibility that `op` is not a `isinstance(op, click.Option)` here?